testing/sdk_networks.py

```diff
@@ -15,6 +16,47 @@
 ENABLE_VIRTUAL_NETWORKS_OPTIONS = {"service": {"virtual_network_enabled": True}}
 
 
+def get_endpoint_names(package_name, service_name) -> list:
+    """Returns a list of endpoint names for the specified service."""
+    rc, stdout, _ = sdk_cmd.svc_cli(package_name, service_name, "endpoints")
+    assert rc == 0, "Failed to get list of endpoints"
+    return jsonlib.loads(stdout)
+
+
+def get_endpoint(package_name, service_name, endpoint_to_get, json=True) -> str:
```

Comment:
I had a similar issue with this function while doing my changes since the return type is ambiguous. It can be Dict and str thus I ended up splitting it up into two functions.